Formula/arduino-cli.rb

```diff
@@ -0,0 +1,18 @@
+class ArduinoCli < Formula
+  desc "Arduino command-line interface"
+  homepage "https://github.com/arduino/arduino-cli"
+  url "https://github.com/arduino/arduino-cli/releases/download/0.5.0-showports/arduino-cli_0.5.0-showports_macOS_64bit.tar.gz"
```

Comment:
This seems to be specific to an OS (macOS). For homebrew this downloads needs to be the full source.